packages/flutter_tools/lib/src/build_system/source.dart

```diff
@@ -99,6 +99,12 @@ class SourceVisitor implements ResolvedFiles {
         // flutter root will not contain a symbolic link.
         Environment.kFlutterRootDirectory => environment.flutterRootDir.absolute.path,
         Environment.kProjectDirectory     => environment.projectDir.resolveSymbolicLinksSync(),
+        Environment.kWorkspaceDirectory   =>
```

Comment:
oh, so the workspace dir is actually the PARENT of `.dart_tool/`?

Reply:
Yeah - I'll try to clarify!